smarts/env/gymnasium/wrappers/api_reversion.py

```diff
@@ -21,7 +21,7 @@
 # THE SOFTWARE.
 from typing import Any, Dict, SupportsFloat, Tuple
 
-import gym
+import gymnasium as gym
 
 
 class Api021Reversion(gym.Wrapper):
```

Comment:
+ I guess we cannot change gym to gymnasium here in the compatibility wrapper, as it is meant to support subsequent layers which use gym interface.
+ We may consider removing this wrapper entirely and let the users build their own compatibility wrapper or use compatibility wrappers provided by RL libraries of their choice.

Reply:
That might be reasonable, although this is just a primitive wrapper. It does not do the full conversion between the two since it still passes up the `gymnasium` spaces which act like `gym` spaces but are not. It is fragile because changes to the `gymnasium.Space` interface would break this wrapper in the future.

Reply:
I have removed it.